src/components/vega-wallet/vega-wallet.tsx

```diff
@@ -389,6 +389,7 @@ const VegaWalletConnected = ({
 
   const disconnect = (
```

Comment:
Can we rename this to footer or something?
```suggestion
  const footer = (
```

Reply:
done